signalfx-tracing/signalfx-go-tracing/gin/server/server.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/signalfx/signalfx-go-tracing/tracing"
+	"github.com/signalfx/tracing-examples/signalfx-tracing/signalfx-go-tracing/gin/server/database"
+	"github.com/signalfx/tracing-examples/signalfx-tracing/signalfx-go-tracing/gin/server/routes"
+)
+
+const (
+	// ServiceName contains name of this service. This will show up on traces
+	ServiceName = "signalfx-battleship"
+	// TracingEndpoint contains a url to send traces
+	TracingEndpoint = "http://localhost:9080/v1/trace"
+	// Port that the app runs
+	Port = 3030
+	// MongoDriver is mongodb driver type. Selections are "mgo" | "mongo"
+	MongoDriver = "mongo"
+	// MongoHost is database hostname
+	MongoHost = "localhost"
+	// MongoPort is database port number
+	MongoPort = 27017
+	// MongoDbName is database name
+	MongoDbName = "bships"
+)
+
+func init() {
+	tracing.Start(tracing.WithEndpointURL(TracingEndpoint), tracing.WithServiceName(ServiceName))
```

Comment:
I think it'd be better to remove the `tracing.WithEndpointURL(TracingEndpoint)` since it's useful to be able to override it with an env var.